src/test/ui/consts/read_from_static_mut_ref.rs

```diff
@@ -1,10 +1,8 @@
-// run-pass
 #![feature(const_mut_refs)]
 #![allow(const_err)]
 
-static OH_YES: &mut i32 = &mut 42;
-
+static OH_NO: &mut i32 = &mut 42;
+//~^ ERROR references in statics may only refer to immutable values
 fn main() {
-    // Make sure `OH_YES` can be read.
-    assert_eq!(*OH_YES, 42);
+    assert_eq!(*OH_NO, 42);
```

Comment:
What purpose does this test serve now?

Reply:
should i remove it?

Reply:
Maybe just leave a note? If we ever go back and allow it in a `static`, it will be nice to have a test in place.